rust/main/utils/run-locally/src/invariants/termination_invariants.rs

```diff
@@ -256,6 +256,15 @@ pub fn scraper_termination_invariants_met(
         return Ok(false);
     }
 
+    fetch_metric(
+        SCRAPER_METRICS_PORT,
+        "hyperlane_contract_sync_stored_events",
+        &hashmap! {"data_type" => "gas_payment"},
+    )?
+    .iter()
+    .map(|s: &String| s.as_str())
+    .for_each(|s| println!("GAS_PAYMENT: {:?}", s));
```

Comment:
It looks unrelated to this change. I wonder why we need this? Does it help with debugging of e2e tests?

Reply:
Yea, I was testing out some stuff. Because it seemed to be passing locally, but not on CI